encoding/codecv2/codecv2.go

```diff
@@ -183,6 +184,15 @@ func ConstructBlobPayload(chunks []*encoding.Chunk, useMockTxData bool) (*kzg484
 		return nil, common.Hash{}, nil, err
 	}
 
+	// Only apply this check when the uncompressed batch data has exceeded 128 KiB.
+	if len(blobBytes) > 131072 {
```

Comment:
This should be `len(batchBytes)` ?

Reply:
nice catch, this is fixed in https://github.com/scroll-tech/da-codec/pull/18. when using the new patch in rollup-relayer, some unit tests failed due to the bug you mentioned.

Reply:
blobBytes here means batchBytes, the naming is outdated.